ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,203 @@
+
+Actions, also known as transitions, are the individual tasks being executed in the workflow and are represented by a rectangle.
+
+Actions can contain any number of input and output ports, which can be mandatory or optional. For detailed information about the types of input/output ports, see [Input and output ports](#input-and-output-ports).
+
+ When adding a new action, all compatible input and output ports are included. Mandatory input ports are already displayed and linked to a place, whereas optional ones are displayed with a ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button.
+
+ All places linked to inputs and outputs ports must be provided with one or more pieces of data, also known as tokens, for the actions to be executed and the workflow to be completed. Otherwise, the workflow stops at the point where tokens are missing. Once an action is executed, one or more output data (tokens) are generated, which in turn become(s) input data (tokens) for the next action. For detailed information about tokens, see [Tokens](#to-be-determined).
+
+When clicking an action, it becomes highlighted and the corresponding element settings are displayed in the settings side bar.
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+
+The settings side bar is located to the right of the workspace and displays the following information:
+
+  - (Action label and name)  
+  The action label can be changed in the *Label* entry field. The action name provides a detailed description of the task to be performed and is read-only.
+
+  - *Key*  
+   Element key. This information is read-only.
+
+  - *Label*  
+  Action label. Click this field to edit the action label. The action label is changed both in the settings side bar and in the diagram.
+
+  - *Queue type*  
+  Click the drop-down list to select the applicable queue type. For detailed information about queue types, see [Configure the queue types](#to-be-completed).
+
+  - *Task event*  
+  Click the drop-down list to select the applicable task event.
+
+  > [Info] This option is linked to the *Tasks* module. It is only displayed if the *Tasks* module is installed and the user has the necessary rights. The *ActindoWorkflow Process Execution Failed* task is preconfigured per default. A list of all existing task events is displayed in the *Events* menu entry of the *Tasks* module.
+
+  - *Configuration*  
+  Depending on the action, there might be configuration options, for instance, defining an email where a receipt or document must be sent. Editing the action configuration requires advanced programming knowledge.
+
+  > [Info] Not all actions have configuration options. In this case, the *Configuration* field is not displayed.
+
+  - *Static inputs*  
+  The optional inputs are displayed. They can be edited by clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button.
```

Comment:
hinzufügen: alle noch nicht verlinkten inputs werden unter den static inputs aufgelistet. Mindestens 1 input muss mit place verbunden sein und darf kein static input sein, damit im process eine action ausgeführt werden kann

Reply:
Gehört diese Info nicht zu den Prozessen: Mindestens 1 input muss mit place verbunden sein und darf kein static input sein, damit im process eine action ausgeführt werden kann. 

Vorschlag: All unlinked inputs are listed in this section. The static inputs determine the data type that can be input. They are preconfigured but can be further configured by the user clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button. Only JSON values are accepted. When being edited, static inputs display a small arc without a place infront of them in the workflow diagram.

[comment]: <> (Unsure! Check Wissentransfer!)